lambda/app/src/controllers/requests.ts

```diff
@@ -278,47 +291,33 @@ export const getRequests = async (session: Session, user: User, include: string
   return requests;
 };
 
-const requestHasBeenMerged = async (id: number) => {
-  try {
-    const request = await models.event.findOne({
-      where: { requestId: id, eventCode: { [Op.in]: [EVENTS.REQUEST_APPLY_SUCCESS, EVENTS.REQUEST_APPLY_FAILURE] } },
-    });
-    if (request) return [true, null];
-    return [false, null];
-  } catch (err) {
-    return [null, err];
-  }
-};
-
 export const deleteRequest = async (session: Session, user: User, id: number) => {
   try {
     const where = getWhereClauseForRequest(session, user, id);
-    const original = await models.request.findOne({ where, raw: true });
+    const current = await models.request.findOne({ where });
```

Comment:
oh, the where is  a variable here? That was a bit confusing for a naming convention. Is that a standard naming convention

Reply:
I see, I think we can use `whereClause` instead of `where` but in that case we need to pass that value as `{ where: whereClause }`.